lib/pronto/config.rb

```diff
@@ -11,6 +11,10 @@ def initialize(config_hash = ConfigFile.new.to_h)
       end
     end
 
+    def consolidate_comments?
+      !@config_hash['consolidate_comments'].nil?
```

Comment:
That's up to @mmozuras to decide. 😀 

Happy to contribute the change if it's something he's keen on.